### PR TITLE
Add RHACM install

### PR DESCRIPTION
**Pull request by akrzos**

### CODE REVIEW

File: ansible/roles/rhacm_hub/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      kind: MultiClusterHub
+      metadata:
+        namespace: open-cluster-management
+        name: multiclusterhub
+      spec: {}
+  register: mch_result
+  until: mch_result.failed == false
+  retries: 60
+  delay: 2
+  ignore_errors: true
```
Comment (smalleni) [inline ansible/roles/rhacm_hub/tasks/main.yml:24]: What's the thought behind this ignore_errors?

Comment (akrzos) [reply ansible/roles/rhacm_hub/tasks/main.yml:24]: Ah, good point it is un-needed. I'll remove it and rebase the pr.

**Review by smalleni — APPROVED** [review]
LGTM

File: ansible/roles/rhacm_hub/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+# Install RHACM hub onto a cluster
+
+- name: Install RHACM
+  k8s:
+    state: present
+    src: "{{ role_path }}/files/rhacm.yml"
+
```
Comment (mukrishn) [inline ansible/roles/rhacm_hub/tasks/main.yml:8]: @akrzos and @smalleni Just to keep it consistent with webscale operator installation tasks https://github.com/redhat-performance/webfuse/blob/main/ansible/roles/operators/tasks/main.yml#L21 
Lets add a task to check if installation was successful ?

Comment (mukrishn) [reply ansible/roles/rhacm_hub/tasks/main.yml:8]: Status check or a `wait_for` task can be added later.

**Review by mukrishn — APPROVED** [review]

**Pull request merged**